app/views/clone_account_memberships/new.html.haml

```diff
@@ -0,0 +1,28 @@
+= content_for :tabnav do
+  = render partial: "admin/shared/tabnav_users", locals: {secondary_tab: "accounts"}
+= content_for :sidebar do
+  = render :partial => 'admin/shared/sidenav_users', locals: { sidenav_tab: 'users' }
+
+= content_for :h1 do
+  = current_facility
+
+= form_with url: facility_user_clone_account_memberships_path(user_to_clone_id: @user_to_clone), html: { id: "cloneAccountForm" } do |f|
```

Comment:
There's a lot of legacy code so it's easy to take advantage of an older style. We've been trying more and more to modernize and one of those is using `js--` as our class names for JS-enhanced functionality as opposed to IDs.